qiskit_ibm_runtime/accounts/account.py

```diff
@@ -205,7 +205,7 @@ def _assert_valid_url(url: str) -> None:
         """Assert that the URL is valid."""
         try:
             urlparse(url)
-        except:
+        except:  # noqa: E722
```

Comment:
It can be nice to write a add a few words after the error code, so a developer knows immediately what's to be ignored by ruff

Reply:
Revised at 8d2a811.

I think we can just count on some developer diligence (ie. just checking in ruff rules for the code if they need to revisit an offending line for some reason), but since these are the first introduction of `noqa` in the codebase, let's give it a try and re-evaluate after a while.